patches/api/0501-Added-PlayerUseBowWithoutProjectileEvent.patch

```diff
@@ -0,0 +1,47 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chaosdave34 <david.emanuel@gmx.de>
+Date: Tue, 26 Nov 2024 17:10:15 +0100
+Subject: [PATCH] Added PlayerUseBowWithoutProjectileEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/player/PlayerUseBowWithoutProjectileEvent.java b/src/main/java/io/papermc/paper/event/player/PlayerUseBowWithoutProjectileEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..7ea392d6accb967a545a720718e33ddbb13023c9
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/player/PlayerUseBowWithoutProjectileEvent.java
+@@ -0,0 +1,35 @@
++package io.papermc.paper.event.player;
++
++import org.bukkit.entity.Player;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.player.PlayerEvent;
++import org.bukkit.inventory.ItemStack;
++import org.jetbrains.annotations.NotNull;
++
++public class PlayerUseBowWithoutProjectileEvent extends PlayerEvent {
++
++    private static final HandlerList HANDLER_LIST = new HandlerList();
++
++    private ItemStack projectile;
++
++    public PlayerUseBowWithoutProjectileEvent(@NotNull final Player player) {
```

Comment:
Remove `@NotNull` as per contributing.md, we use jspecify annotations in new paper types.